contracts/upgradeable_contracts/erc20_to_native/ForeignBridgeErcToNative.sol

```diff
@@ -71,7 +79,13 @@ contract ForeignBridgeErcToNative is BasicForeignBridge, ERC20Bridge, OtherSideB
     ) internal returns (bool) {
         setTotalExecutedPerDay(getCurrentDay(), totalExecutedPerDay(getCurrentDay()).add(_amount));
         uint256 amount = _amount.div(10**decimalShift());
-        return erc20token().transfer(_recipient, amount);
+        bool res = erc20token().transfer(_recipient, amount);
+
+        if (AddressUtils.isContract(halfDuplexErc20token()) && tokenBalance(halfDuplexErc20token()) > 0) {
```

Comment:
do you think it make sense to check if it is a contract for the case when we hardcode the return value of the method `halfDuplexErc20token`? When we hardcode the value we know that it is the token contract.

Reply:
You are correct, the check is not needed. Fixed https://github.com/poanetwork/tokenbridge-contracts/pull/319/commits/055bcb62a794ed913dfec10b6558e3b57a314151